api/v01/api_base.rb

```diff
@@ -52,7 +52,7 @@ def self.count_route_locations(params)
       def self.count_matrix_locations(params)
         src_size = count_locations(params[:src])
         dst_size = params[:dst] ? count_locations(params[:dst]) : src_size
-        src_size * dst_size
```

Comment:
For clustering purpose we may need rectangular matrices for exemple 2 (depots) * 5000 (items) before effectively. I was satisfied yesterday by the Math.sqrt(src, dst) could we return to this solution ?